pystiche_papers/ulyanov_et_al_2016/_modules.py

```diff
@@ -120,6 +158,22 @@ def __init__(
 
 
 class ConvSequence(SequentialWithOutChannels):
+    r"""Sequence of convolutional blocks that occurs repeatedly in :cite:`ULVL2016`.
```

Comment:
Since this not mentioned in the paper, please explain what each sequence comprises.